test/core/security/tls_security_connector_test.cc

```diff
@@ -737,6 +737,44 @@ TEST_F(TlsSecurityConnectorTest,
                             on_peer_checked);
 }
 
+TEST_F(TlsSecurityConnectorTest,
+       ChannelSecurityConnectorWithVerifiedRootCertSubjectSucceeds) {
+  auto* sync_verifier_ = new SyncExternalVerifier(true);
```

Comment:
nit: The `_` suffix should not be used on this local variable, only for class member fields.

Reply:
My bad, fixed